app/controllers/collections_controller.rb

```diff
@@ -137,6 +139,8 @@ def web_create_collection
     @collection_title = params[:collection_title]
     @collection_owner = params[:collection_owner]
     @collection_abstract = params[:collection_abstract]
+    @approval_required = params[:approval_required]
+    @approval_required = 'private' if request.get?
```

Comment:
This seems a little funny, how come you need both `GET` and `POST` routes to this method, shouldn't creates only be `POST`ed?